Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy.swift

```diff
@@ -115,7 +115,7 @@ struct PathHierarchy {
             }
             
             var topLevelCandidates = nodes
-            for relationship in graph.relationships where [.memberOf, .requirementOf, .optionalRequirementOf].contains(relationship.kind) {
+            for relationship in graph.relationships where [.memberOf, .requirementOf, .optionalRequirementOf, .extensionTo, .declaredIn].contains(relationship.kind) {
```

Comment:
Nit: Maybe we should have an extension on Symbol graph's relationship kind that consolidates this?

I don't think we're actually duplicating this but it might better convey the intention at least.

```swift
extension RelationshipKind {
    var isChildLike: Bool {
        ...
```